jest-setup.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-undef */
+import {NativeModules} from 'react-native';
+jest.mock('react-native-permissions', () =>
```

Comment:
do we still need this since we're excluding the permissions page?

Reply:
Leaving in because we will need it once I figure out the permissions page issue!